app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class TicketsController < ApplicationController
+  before_action :set_ticket, only: %i[show update]
+  before_action :set_participating_seller, only: %i[create]
+
+  # GET /tickets
+  def index
+    @tickets = Ticket.all
+
+    render json: @tickets
+  end
+
+  # GET /tickets/1
+  def show
+    render json: @ticket
+  end
+
+  # POST /tickets
+  def create
+    puts(params[:number_of_tickets])
+    savedCorrectly = true
+    createdTickets = []
+
+    (1..params[:number_of_tickets]).each do |_i|
+      attributes = {}
+      attributes[:ticket_id] = SecureRandom.alphanumeric(5).upcase.insert(4, '-')
```

Comment:
Will they have to enter the dash when they enter it into the passport app? Should we make them? cc @nanxiy

Reply:
The reason why I bring this up is because if we don't expect people to type the dash in themselves, then we shouldn't store the dash on the DB. If we do expect people to type the dash in themselves, then we can go ahead and add it here like it is

Reply:
i think we shouldn't make them enter the dash, similar to how phone number fields plug in the dashes for you

Reply:
FYI @ath-chen is it possible to automatically show the dash in our input? and if they type the dash on the frontend, just strip it before you send it back?

Reply:
@justintmckibben yeah i can do that, are codes always going to be in a XYZ-123 format (none will be longer / shorter)? To clarify, when it goes to the backend, it'll just look like XYZ123 right?

Reply:
I personally think if we do 5 digits, it should be A1C-J1 and if we do 6, then NWK-3JF. What do you think @nanxiy @jacoblb64 

Three digits at a time is easy to remember when typing things in or reading it off to a friend

Reply:
@ath-chen FYI this would also mean you'd need to manually input the dash when generating the tickets

Reply:
@justintmckibben, I agree with the way you've broken things up, happy with either 5 or 6 digits. I'm pretty sure we're saving with the dash for gift cards, would we want to do the same or different here?

Reply:
Let's not save the dashes here — and migrate the gift cards in the future to strip the dashes and hashes. Poor design decision by 3 months ago me 🙃

Reply:
OK! Going with 5 digits, no dash in the DB everyone!